assets/js/components/date-range-selector.js

```diff
@@ -89,20 +89,20 @@ class DateRangeSelector extends Component {
 
 		// Update this component.
 		this.setState( {
-			dateValue: applyFilters( this.dateRangeHook, __( 'Last 28 days', 'google-site-kit' ) ),
+			dateValue: applyFilters( this.dateRangeHook, 'Last 28 days' ),
 		} );
 
 		return false;
 	}
 
 	render() {
 		const { dateValue } = this.state;
-		const options = [
-			__( 'Last 7 days', 'google-site-kit' ),
-			__( 'Last 14 days', 'google-site-kit' ),
-			__( 'Last 28 days', 'google-site-kit' ),
-			__( 'Last 90 days', 'google-site-kit' ),
-		];
+		const options = {
+			'Last 7 days': __( 'Last 7 days', 'google-site-kit' ),
+			'Last 14 days': __( 'Last 14 days', 'google-site-kit' ),
+			'Last 28 days': __( 'Last 28 days', 'google-site-kit' ),
+			'Last 90 days': __( 'Last 90 days', 'google-site-kit' ),
```

Comment:
These values must be slugs, not "user-friendly" strings, they should be directly passable to the REST API (like `last-28-days` etc.).